src/Native.re

```diff
@@ -99,6 +99,7 @@ module NavigationContainer = {
   [@bs.module "@react-navigation/native"] [@react.component]
   external make:
     (
+      ~ref: ReactNative.NativeElement.ref=?,
```

Comment:
This one seems unecessary

Reply:
It doesn't work without it, and I looked on other project to do it : https://github.com/reason-react-native/datetimepicker/blob/f479766c79ce24248c8752c2b1a6e5edb35bb2b1/src/ReactNativeDateTimePicker.re#L21